src/components/NewCommentForm.tsx

```diff
@@ -12,26 +72,32 @@ export const NewCommentForm: React.FC = () => {
           <input
             type="text"
             name="name"
+            value={nameInput}
+            onChange={evemt => setNameInput(evemt.target.value)}
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={`input ${nameError && 'is-danger'}`}
```

Comment:
Why not use `classnames` or a similar library?

Reply:
there is no hard logic so I don't think I need it in this case)